### PR TITLE
[Table/Create/240323Part2] - Create Table

### PR DESCRIPTION
**Pull request by shun2134**
・Create Table (Migration)
- customer_address_table.php
- address_table.php
- countries_table.php
- payments_table.php
- payment_types_table.php
- products_table.php

* Please refter to our ERD below.
https://dbdiagram.io/d/GI-Mall-or-E-Commerce-Site-65ce1489ac844320ae38a00e

### CODE REVIEW

File: database/migrations/2024_03_23_070921_create_address_table.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    {
+        Schema::create('address', function (Blueprint $table) {
+            $table->id();
+            $table->string('unit_number')->nullable();
+            $table->string('street_number')->nullable();
+            $table->string('address_line1');
+            $table->string('address_line2')->nullable();
+            $table->string('city');
+            $table->string('region');
+            $table->integer('postal_code');
```
Comment (Ninit29) [inline database/migrations/2024_03_23_070921_create_address_table.php]: use string / varchar for postal code

File: database/migrations/2024_03_23_070921_create_address_table.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            $table->string('country_code');
```
Comment (Ninit29) [inline database/migrations/2024_03_23_070921_create_address_table.php]: specify length for country_code

File: database/migrations/2024_03_23_074733_create_products_table.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->integer('price');
```
Comment (Ninit29) [inline database/migrations/2024_03_23_074733_create_products_table.php]: price should be double to cater those with decimal places like 5.75

**Review by shun2134 — COMMENTED** [review]
・Modify
- create_address_table.php : modify "postal_code" and "country_code(foreign key)"
- create_countries_table.php : specify "alpha3" of length
- create_products_table.php : modify "price" * use double

**Review by KobeSergio — APPROVED** [review]

**Pull request merged**